README.md

```diff
@@ -75,7 +75,7 @@ overlay network between them - the `overlay` example topology described below.
 To interact with the cluster, run:
```

Comment:
XR seems to not like using "root" as a username, so the example should probably be something else.